lib/analysis_options.yaml

```diff
@@ -56,25 +59,34 @@ linter:
     annotate_overrides: true # flutter
     avoid_annotating_with_dynamic: true
     avoid_bool_literals_in_conditional_expressions: true
+    avoid_catches_without_on_clauses: true
     avoid_catching_errors: true
     avoid_classes_with_only_static_members: true
+    avoid_equals_and_hash_code_on_mutable_classes: true
     avoid_field_initializers_in_const_classes: true
     avoid_final_parameters: true
     avoid_function_literals_in_foreach_calls: true # flutter
+    avoid_implementing_value_types: true
     avoid_init_to_null: true # flutter
     avoid_multiple_declarations_per_line: true
     avoid_null_checks_in_equality_operators: true # flutter
     avoid_positional_boolean_parameters: true
+    avoid_private_typedef_functions: true
+    avoid_redundant_argument_values: false
     avoid_renaming_method_parameters: true # flutter
     avoid_return_types_on_setters: true # flutter
     avoid_returning_null_for_void: true # flutter
+    avoid_returning_this: true
     avoid_shadowing_type_parameters: true # core
     avoid_single_cascade_in_expression_statements: true # flutter
+    avoid_types_on_closure_parameters: false
     avoid_unnecessary_containers: true # flutter
     avoid_unused_constructor_parameters: true
     avoid_void_async: true
     camel_case_extensions: true # core
     camel_case_types: true # core
+    cascade_invocations: false
```

Comment:
Why?

Reply:
for readability it might sometimes be better to not cascade.

Reply:
Really? I find cascading better readable, that's why it exists in the first place.